src/configs/constants.js

```diff
@@ -0,0 +1,7 @@
+export const PUBLIC_IMAGE_FOLDER = '/images/';
+
+export const DEFAULT_BANNER_IMAGE = 'banners/default.png';
+
+export const banners = ['banners/cloud.jpg', 'banners/dns-server.png', 'banners/full-stack-web-development.jpg', 'banners/load-balancer.png', 'banners/js.jpg'];
+
+export const total = 6;
```

Comment:
Why use hard-code value for total number of banner 
Use the banners.length where required